bedrock/firefox/templates/firefox/privacy_tour/index.html

```diff
@@ -11,6 +11,20 @@
 
 {% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
 
+{% block page_title %}
```

Comment:
I'm a bit confused as to why we're updating strings on this page? The privacy tour was shown as a whatsnew page for users updating to 33.1, which shipped quite a while ago. I doubt there's much traffic to this page (if any at all). It's also a page only viewable by FF users and is not indexed by search engines, so I'm not sure what benefit there is here.